apps/docs/content/docs/components/avatar.mdx

```diff
@@ -279,17 +279,21 @@ import { Avatar } from '@nextui-org/react';
 | **zoomed**       | `boolean`               | `true/false`                             | Zoomed avatar                          | `false`   |
 | **squared**      | `boolean`               | `true/false`                             | Squared avatar                         | `false`   |
 | **icon**         | `ReactNode`             | -                                        | Show icon in avatar                    | -         |
+| **css**          | `styleObject`           | `css.properties`                         | Override Default CSS style             | -         |
```

Comment:
Hey @Dhananjay-JSR thanks for this PR could you please change all `css` prop `type` from `styledObject` to `Stitches.CSS` and the Accepted values to `-`?